src/stores/pacbioLibraries.js

```diff
@@ -155,47 +121,28 @@ export const usePacbioLibrariesStore = defineStore('pacbioLibraries', {
      * @param {number} page - The page number to fetch from the server.
      * @returns {Promise<{success: boolean, errors: Array}>} - A promise that resolves to an object containing a success boolean and an array of errors.
      */
-    async fetchLibraries(filter = {}, page = {}) {
-      const rootStore = useRootStore()
-      const pacbioLibraries = rootStore.api.traction.pacbio.libraries
-      const promise = pacbioLibraries.get({
-        page,
-        filter,
-        include: 'request,tag,tube',
-      })
-      const response = await handleResponse(promise)
-
-      const { success, body: { data, included = [], meta = {} } = {}, errors = [] } = response
-
+    async fetchLibraries(filterOptions) {
+      const { success, data, meta, errors, libraries, tubes, tags, requests } =
+        await fetchLibraries({
```

Comment:
I think this naming should be changed so its clear this isnt a recursive function.